### PR TITLE
.Net: Fix GitHub Repo Q&A Bot Sample

### PR DESCRIPTION
**Pull request by crickman**
### Motivation and Context
Not returning relevant results: [GitHub Repo Q&A Bot Sample](https://github.com/microsoft/semantic-kernel/tree/main/samples/apps/github-qna-webapp-react)

Customer reported: https://github.com/microsoft/semantic-kernel/issues/2506

### Description
RCA: Mistmatch between container name being written to and being queried.

`https://api.github.com/repos/microsoft/semantic-kernel-main`

vs

`https://github.com/microsoft/semantic-kernel-main`

image
image

### Contribution Checklist
- [x] The code builds clean without any errors or warnings
- [x] The PR follows the [SK Contribution Guidelines](https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md) and the [pre-submission formatting script](https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md#development-scripts) raises no violations
- [x] All unit tests pass, and I have added new tests where possible
- [x] I didn't break anyone :smile:

### CODE REVIEW

File: samples/dotnet/KernelHttpServer/Plugins/GitHubPlugin.cs [hunk 1]
```diff
@@ -213,8 +214,6 @@ private async Task SummarizeCodeDirectoryAsync(string directoryPath, string sear
 
         if (filePaths != null && filePaths.Length > 0)
         {
-            this._logger.LogDebug("Found {0} files to summarize", filePaths.Length);
```
Comment (markwallace-microsoft) [inline samples/dotnet/KernelHttpServer/Plugins/GitHubPlugin.cs:216]: Why remove this, it seems helpful?

**Review by markwallace-microsoft — APPROVED** [review]
LGTM just one comment

File: samples/dotnet/KernelHttpServer/Plugins/GitHubPlugin.cs (hunk 1 above)
Comment (crickman) [reply samples/dotnet/KernelHttpServer/Plugins/GitHubPlugin.cs:216]: Accidental destructive debugging...I thought I'd rolled any of this back.  Thanks for the callout.

File: samples/dotnet/KernelHttpServer/Plugins/GitHubPlugin.cs [hunk 2]
```diff
@@ -196,10 +197,10 @@ private async Task SummarizeCodeFileAsync(string filePath, string repositoryUri,
...
             {
                 await this._kernel.Memory.SaveInformationAsync(
-                    $"{repositoryUri}-{repositoryBranch}",
-                    text: $"{code} File:{repositoryUri}/blob/{repositoryBranch}/{fileUri}",
-                    id: fileUri,
-                    cancellationToken: cancellationToken);
+                     $"{repositoryUri}-{repositoryBranch}",
+                     text: $"{code} File:{repositoryUri}/blob/{repositoryBranch}/{fileUri}",
+                     id: fileUri,
+                     cancellationToken: cancellationToken);
```
Comment (gitri-ms) [inline samples/dotnet/KernelHttpServer/Plugins/GitHubPlugin.cs:203]: nit: get rid of the extra space at the beginning of each line? (unless that's an intentional spacing fix)

**Review by gitri-ms — APPROVED** [review]

File: samples/dotnet/KernelHttpServer/Plugins/GitHubPlugin.cs (hunk 2 above)
Comment (crickman) [reply samples/dotnet/KernelHttpServer/Plugins/GitHubPlugin.cs:203]: I thought visual studio was fix bading spacing...either way, I didn't mean to do this.  I've repeatedly auto-formatted...manually fixing.

**Review by TaoChenOSU — APPROVED** [review]

**Pull request merged**